operations/restrictions.md

```diff
@@ -468,7 +468,106 @@ This reduces the overall number of restrictions and improves system performance.
 | `MinPrice` | [Currency value](accountingitems.md#currency-value)| optional | Value of the minimum price per time unit. |
 | `MaxPrice` | [Currency value](accountingitems.md#currency-value)| optional | Value of the maximum price per time unit. |
 
+## Clear restrictions
+
+> ### Restricted!
+> This operation is currently in beta-test and as such it is subject to change. Use of this operation must be enabled per enterprise. Please contact the Technical Partner Support team in order to enable it.
+
+Deletes restrictions that [match the conditions](#matching-conditions) using the [splicing algorithm](#splicing-algorithm).
+
+This operation is intended to be used in conjunction with [Set restrictions](#set-restrictions).
+
+### Matching conditions
+
+The [spliced](#splicing-algorithm) restriction must match the specified conditions exactly. Consider the following example:
+
+Service has restrictions A and B. Restriction A applies to resource category C1 and rate R1. Restriction B applies to resource category C1 to all rates.
+
+If the clear data specified have specified resource category C1 but do not specify any rate, only restriction B is spliced. Restriction B remains untouched, even though it is applied to resource category C1 as well.
```

Comment:
```suggestion
If the clear data specified have specified resource category C1 but do not specify any rate, only restriction B is spliced. Restriction A remains untouched, even though it is applied to resource category C1 as well.
```

Isn't Restriction A untouched because Restriction A applies to a specific resource AND rate? Or am I understanding the logic incorrectly? :)